src/browser/BrowserAccessControlDialog.cpp

```diff
@@ -38,9 +39,18 @@ BrowserAccessControlDialog::~BrowserAccessControlDialog()
 
 void BrowserAccessControlDialog::setUrl(const QString& url)
 {
-    m_ui->label->setText(QString(tr("%1 has requested access to passwords for the following item(s).\n"
+    QString dialogText;
+    if (m_httpAuth) {
+        dialogText = QString(tr("%1 has requested HTTP Basic Auth access to passwords for the following item(s).\n"
```

Comment:
The term "HTTP Basic Auth" means something to a very small percentage of people. What is the significance of this request over just asking for permission to my passwords? Why am I getting all these popups? (These are the questions that will go through normal people's minds).